organization/operations.md

```diff
@@ -11,81 +11,56 @@ if approved by NumFOCUS board.
 
 ## NumFOCUS Administrator for GSoC
 
-The NumFOCUS board will indicate one person,
+The NumFOCUS board will indicate two persons,
 named in this document as **organization administrator**,
 to be in charge of NumFOCUS application **for each edition** of GSoC.
 
-The **organization administrator** is responsible for
-
--   create the initial application for NumFOCUS to apply for GSoC
--   advertise the application to **all** projects/organization supported
-    by NumFOCUS
--   manage NumFOCUS profile at GSoC
--   request a minimum and a maximum number of slots
-    that can accomodate the slots requested by each project/organization
-    under NumFOCUS umbrella.
-
-## Coordinators for GSoC
-
-Each project/organization under NumFOCUS umbrella for GSoC
-can indicate up to three names,
-named in this document as **coordinators**,
-to help the **organization administrator** managing NumFOCUS application.
-
-**Coordinators** will have administrative access to
-NumFOCUS profile at GSoC.
-
-The **coordinators** are responsible for
-
--   provide a list of projects ideas to **organization administrator**
--   advertise the application to the project/organization they represent
--   connect users with NumFOCUS profile
--   request to **organization administrator** a minimum and a maximum number of
-    slots
-
-## Scoring Students Proposals
-
-Students proposals will be scoring following the description bellow.
-
--   **0 stars**: must be reject.
--   **1 star**: slot should be request and should be reject if not enough slots are received.
--   **2 stars**: slot should be request
--   **3 stars**: slot should be request and should be accepted even if few slots are received.
-
-**Coordinators** and **mentors** should vote **only** in proposals
-related with the project/organization they are involved.
-
-**Organization administrator** will request one slot for every
-proposal with at least one star.
+The **organization administrators** are responsible for
+
+- create the initial application for NumFOCUS to apply for GSoC
+- advertise the application to **all** projects/organization supported
+  by NumFOCUS
+- manage NumFOCUS profile at GSoC
+- request a minimum and a maximum number of slots
+  that can accomodate the slots requested by each project/organization
+  under NumFOCUS umbrella.
+- stay in contact with sub-org admins
+
+### How to sign up mentors
+
+You can use google forms to sign up mentors who want to work with NumFOCUS. This
+will have the advantage that we automatically have a list of all mentors and
+their contact information. Mentors should be given access to numfocus mentoring
+mailing list.
+
+## Timeline for Organization Administrators
+
+*January*: Ask NumFOCUS if they want to participate. Preferably with help of NumFOCUS staff
```

Comment:
@kain88-de 
Once we have the go ahead from NumFOCUS we need to have a decent amount of project ideas to make our GSoC application strong. I suggest we ask sub orgs (admins) who have regularly participated in GSoC to help us with the ideas page.

- Start preparing an ideas page with the sub orgs admins.